src/Servers/IIS/IntegrationTesting.IIS/src/IISDeploymentResult.cs

```diff
@@ -43,5 +43,10 @@ public HttpClient CreateClient(HttpMessageHandler messageHandler)
         };
     }
 
+    public void Dispose()
+    {
+        HttpClient.Dispose();
```

Comment:
Weren't there issues with this when Hao tried it before?

Reply:
We finally got confirmation that this issue is a bug that's been fixed in a future version of IIS/Windows. I don't recall any issues with the mitigation?